the-example-app/README.md

```diff
@@ -1,36 +1,31 @@
-What is `The Example App model`?
+Content model: `The Example App`
 ================================
 
 The example app content model containts the model for all the example apps, demonstrating in a broader fashion some examples on how to use
-Contentful. If you want to see the model in action, we do have a [hosted version](https://contentful-example-app.herokuapp.com/), for you
+Contentful. If you want to see the model in action, we do have a [hosted version](https://the-example-app-nodejs.herokuapp.com/), for you
 to check out.
 
-Through our [command line app](https://github.com/contentful/contentful-cli) you can push this model to your own space. Doing so will be
+Through our [command line app](https://github.com/contentful/contentful-cli) you can import this model into your own space. Doing so will be
 accomplished by this call to the command line app:
 
 ```
 contentful space seed the-example-app
 ```
 
-The app will walk you through the steps needed in order to fill your space.
+The Contentful CLI will walk you through the steps needed in order to import this content model into your space.
 
-Other apps
-==========
-
-The Node example app is not the only one. Soon™ there will be more available, all using the
-same space.
-
-Stay tuned for updates on the model and example apps.
-
-
-
-Credentials
+Versions
 ===========
 
-**⚠️WARNING: The following keys are just for demonstration purposes. Normally you would store them on a save place somwhere in the app, but for the demo, feel free to use them :WARNING⚠️**
+We are hosting versions of this content model for demo purposes. `The Example App` can be connected to these versions through `variables.env` when run locally or through query paramters when hosted. For example: https://the-example-app-nodejs.herokuapp.com/?space_id=<YOUR_CLONED_SPACE_ID>&delivery_token=<YOUR_DELIVERY_TOKEN>&preview_token=<YOUR_PREVIEW_TOKEN>
```

Comment:
I would consider adding "different" as in "We are hosting different versions..." not quite sure if it's necessary, but I feel it reads better.